salt/runners/virt.py

```diff
@@ -217,6 +218,9 @@ def init(
 
     saltenv
         The Salt environment to use
+
+    seed_cmd
+        The command to use if 'seed=True' (default: 'seed.apply').
```

Comment:
Can you add a `.. versionadded:: Fluorine` tag to the docs for this new option?

Reply:
Absolutely.